### DISCUSSION TITLE
ModbusTCPConnect

### DISCUSSION THREAD

**Discussion by mbu10** [arendst/Tasmota#22396] [General]
so i have eventually got the modbustcp working ( i presume that you start the port on one side ) and then the other connects which it does 
"19:29:15.732 TCP: MBS: MBRTCP Connecting to 192.168.2.100 on port 500
19:29:15.862 TCP: MBS: MBRTCP connected!
19:29:15.864 RSL: RESULT = {"ModbusTCPConnect":"Done"}

now bothT-CAN485
have the rs485 rx 21  - TX 22
in the module i have selected 

MODBR RX pin 21
MODBR TX pin 22
 for both sides 
but there is no information , now i programmed another lilly and that had some esp code(manafactuers code)  and it pulled the info from the SDM  ,  so i know the cables are all the right way round and the device can pull it 
slightly lost now 

this is the picture of the esp i am picking the right ports ?

https://github.com/Xinyuan-LilyGO/T-CAN485

![T-CAN485-PINMAP1-EN](https://github.com/user-attachments/assets/135819e9-a42a-4fd9-9a3d-740f17aa5d5f)

**Comment by tanelvakker** [comment]
<img width="341" alt="Screenshot 2025-01-02 at 20 33 10" src="https://github.com/user-attachments/assets/5de79313-f0ba-4ffc-ad9d-e170e7d2f595" />
Got it working with these settings
